next/content/docs/guides/vercel.md

```diff
@@ -1,73 +1,82 @@
 ---
-title: Connect a Next.js application to Neon
+title: Connect from Vercel to Neon
```

Comment:
this is a mutual integration we also get some data from vercel, so I'd rather phrase it: 'Connect Vercel and Neon'

Reply:
Changed as suggested